grid/app/main/codes.py

```diff
@@ -25,11 +25,11 @@ class WEBRTC_EVENTS(object):
     JOIN_ROOM = "webrtc: join-room"
 
 
-class FL_EVENTS(object):
-    HOST_FL_TRAINING = "federated/host-training"
-    REPORT = "federated/report"
-    AUTHENTICATE = "federated/authenticate"
-    CYCLE_REQUEST = "federated/cycle-request"
+class STATIC_FL_EVENTS(object):
+    HOST_FL_TRAINING = "static/federated/host-training"
```

Comment:
We can get rid of the federated part for all these resources.

Reply:
@cereallarceny If we remove the federated from /static/federated/, then we might clash with the static files routes.

Reply:
See my comment reply below.